### ISSUE TITLE
[YT-DL] Can't download video anymore.

### ISSUE THREAD

**Issue by ArturiaPendragon** [skid9000/Crunchyroll-Downloader#50]
**Checklist**

- [x] I checked if my issue is already referenced.
- [x] I use a release version (from a tag or master branch).
- [x] Youtube-DL is up to date
- [x] I use the version of .NET Framework that is recommended or better

**Describe the bug**

**To Reproduce**

'Login seems to be successfull'
'Add video URL'
'check subtitle and choose the subtitle - mkv is unchecked'
See error

**Expected behavior**
When I clic on "Start" I get an error: Something wrong has happened while downloading the video. Please check your URL of login credentials.
For information, all is working fine few days / last weeks

**Screenshots**
![img111_x8 (2)](https://user-images.githubusercontent.com/63142830/80513450-5c437c80-897f-11ea-8fd3-9cbb566926d6.png)

**Infos (please complete the following information):**
OS (from a cmd : winver) : Win7 X64
Youtube-DL version (from a cmd : "C:\ProgramData\Crunchy-DL\youtube-dl.exe --version" 2020.05.08
Crunchyroll-Downloader version (click on the about button from MainWindow) : 2.1.1
Region of the Crunchyroll account : FR
URL of what you tried to download : any video

**Comment by skid9000** [comment]
Yep, it seems like CloudFlare put this stupid JsChallenge crap back on the login page...
Will see what we can do at the youtube-dl level.

https://github.com/ytdl-org/youtube-dl/issues/25437

**Comment by werewolf004** [comment]
Is there some news about this? I have checked on yt-dl but I don't see anything, I try with the lastest published version of yt-dl but always the same problem

**Comment by deathzombee** [comment]
This may help https://github.com/Anorov/cloudflare-scrape.git
or more importantly this pull to yt-dl https://github.com/ytdl-org/youtube-dl/pull/20528

**Comment by skid9000** [comment]
https://github.com/ytdl-org/youtube-dl/issues/25437#issuecomment-636099161

**Comment by zerobkpro** [comment]
Following the error instructions will get resolved or there is no method for the error yet?

**Comment by skid9000** [comment]
The issue is on youtube-dl end, i can't do anything until a patch exist.
